src/components/IncorporationAgreement/AgreementType.vue

```diff
@@ -129,6 +130,12 @@ export default class AgreementType extends Vue {
     this.agreementType = this.getAgreementType
     this.setIncorporationAgreementData()
   }
+
+  /** Updates store when local Editing property has changed. */
+  @Watch('showAgreementTypeForm', { immediate: true })
```

Comment:
I thought this was more modular than replacing the local property with the store property.